dev/service-registry/client-javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/registry/RegisteredService.java

```diff
@@ -26,9 +22,7 @@
  * {@code RegisteredService.of()}.
  */
 @SuppressWarnings("all")
-@ParametersAreNonnullByDefault
 @Generated({"Immutables.generator", "AbstractRegisteredService"})
```

Comment:
> To be honest, I think we should replace these classes with hand written ones.

That sounds great.

I didn't realize that the `javax.annotation` package is split between the standard library and other jars! That seems like a really bad idea. I think Jigsaw might even disallow it.